docs/sphinx/source/whatsnew/v0.11.2.rst

```diff
@@ -15,6 +15,10 @@ Enhancements
 Documentation
 ~~~~~~~~~~~~~
 * Edited docstrings for :py:func:`~pvlib.pvsystem.dc_ohms_from_percent` and
+  :py:func:`~pvlib.pvsystem.dc_ohmic_loss` for clarity. (:issue:`1601`, :pull:`2229`)
+* Added a new reference style guide to the contributing page.
```

Comment:
```suggestion
* Added a new bibliographic reference style guide to the contributing page.
```
Plain `reference style guide`, reads like a style guide of reference for code style? IDK, it's good as it is.
May as well link to it with ```:ref:`references` ```.

Reply:
```suggestion
* Added a new reference citation style guide to the contributing page.
```
I think "citation" is more clear than "bibliographic"

Reply:
> I think "citation" is more clear than "bibliographic"

Agreed

But then "reference citation style guide" as per the [suggestion](https://github.com/pvlib/pvlib-python/pull/2226#discussion_r1797250422), or just "citation style guide"? I lean towards the latter

Reply:
Latter is simpler, I agree